apps/site/src/app/apply/sections/Form/ResumeInformation.tsx

```diff
@@ -67,9 +68,25 @@ export default function ResumeInformation() {
 			</label>
```

Comment:
Issue: If we have made the resume optional, then we should remove the asterisk in the label